deploy/layered-sre-authorization/01-layered-cs-sre-admin.ClusterRole.yaml

```diff
@@ -0,0 +1,87 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: layered-cs-sre-admin
+rules:
+# CS SRE can get, list, and watch all resources
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - get
+  - list
+  - watch
+ # CS SRE can run commands in pods
+- apiGroups:
+  - ""
+  resources:
+  - pods/exec
+  verbs:
+  - create
+# CS SRE can delete jobs and builds
+- apiGroups:
+  - batch
+  resources:
+  - jobs
+  verbs:
+  - delete
+  - deletecollection
+- apiGroups:
+  - build.openshift.io
+  resources:
+  - builds
+  verbs:
+  - delete
+  - deletecollection
+# CS SRE can manage projects
+- apiGroups:
+  - config.openshift.io
+  resources:
+  - projects
+  verbs:
+  - '*'
+- apiGroups:
+  - project.openshift.io
+  resources:
+  - projects
+  verbs:
+  - '*'
+# CS SRE can manage namespaces
+- apiGroups:
+  - ""
+  resources:
+  - namespaces
+  - namespaces/finalize
+  verbs:
+  - '*'
+# CS SRE can use oc debug
```

Comment:
Using `oc debug` requires being run in the `default` namespace.  Need to split this out to a separate permissions.
See https://github.com/openshift/ops-sop/blob/b40edcb9c761d7023b7a8ced30cc9c7aea05cad1/v4/support.asciidoc#23-access-node-via-oc

Reply:
The intention for this is to run debug containers in RHMI namespaces and not against nodes. Not sure if I still need to split out. If I do, what permissions do I need to split this to? CS SRE have access to the default NS.

Reply:
Good point, I think "node" when I see `oc debug`.  Appears to work on a pod, so yes this is fine as-is.